test

```diff
@@ -13,6 +13,7 @@ source ./build
 
 TESTABLE="plugins/ipam/dhcp plugins/main/loopback pkg/invoke pkg/ns pkg/skel"
 FORMATTABLE="$TESTABLE libcni pkg/ip pkg/ns pkg/types pkg/ipam plugins/ipam/host-local plugins/main/bridge plugins/meta/flannel plugins/meta/tuning"
+LINT="./..."
```

Comment:
I think it's not necessary to lint-check the _vendor_ directory. Maybe checking `$FORMATTABLE` would be enough